endoreg_db/services/video_import.py

```
@@ -0,0 +1,307 @@
+
+def _ensure_frame_cleaning_available():
+    """
+    Ensure frame cleaning modules are available by adding lx-anonymizer to path.
+    
+    Returns:
+        Tuple of (availability_flag, FrameCleaner_class, ReportReader_class)
+    """
+    try:
+        # Check if we can find the lx-anonymizer directory
+        current_file = Path(__file__)
+        endoreg_db_root = current_file.parent.parent.parent.parent
```

Ruff (0.12.2)

Local variable `endoreg_db_root` is assigned to but never used

Remove assignment to unused variable `endoreg_db_root`

(F841)